dask/array/tests/test_xarray.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+import dask.array as da
+from ..utils import assert_eq
+
+xr = pytest.importorskip('xarray')
+
+
+def test_xarray():
+    y = da.mean(xr.DataArray([1, 2, 3.0]))
+
+    assert_eq(y, y)
```

Comment:
does this verify that the type of the result is a dask array? `assert isinstance(y, da.Array)` might be a more direct way to do this.

Reply:
Well, I could imagine future behavior being that the result here is an Xarray thing rather than a Dask thing.  I actually don't care too much about the output, as long as it computes and is self-consistent.